src/main/java/seedu/duke/trackables/Event.java

```diff
@@ -0,0 +1,16 @@
+package seedu.duke.trackables;
+
+public class Event extends Task {
+
+    protected String at;
```

Comment:
Maybe you want to use another name such as "time" instead of "at" for the string which stores the date? I think this would improve the clarity of your code.